src/test/TestUtils.h

```diff
@@ -30,10 +30,9 @@ template <typename T = ApplicationImpl>
 std::shared_ptr<T>
 createTestApplication(VirtualClock& clock, Config const& cfg)
 {
-    auto app = Application::create<T>(clock, cfg);
-    auto& lm = app->getLedgerManager();
-    lm.getCurrentLedgerHeader().baseFee = cfg.DESIRED_BASE_FEE;
-    testutil::setCurrentLedgerVersion(lm, cfg.LEDGER_PROTOCOL_VERSION);
+    Config c2(cfg);
```

Comment:
I generally dislike this pattern. `cfg` is being passed by const-reference then immediately copied in the function body. I think it would be better to let `cfg` be passed by value, and remove `c2` entirely. Better to let the compiler do the work for you, in my opinion. (See the next comment before you fix this though)